activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/main/java/org/activiti/cloud/services/rest/controllers/ProcessVariablesHelper.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.activiti.cloud.services.rest.controllers;
+
+import java.text.MessageFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.TimeZone;
+
+import org.activiti.api.process.model.payloads.SetProcessVariablesPayload;
+import org.activiti.engine.ActivitiException;
+import org.activiti.spring.process.model.Extension;
+import org.activiti.spring.process.model.ProcessExtensionModel;
+import org.activiti.spring.process.model.VariableDefinition;
+import org.activiti.spring.process.variable.VariableValidationService;
+
+public class ProcessVariablesHelper  {
```

Comment:
can we give a different name instead of "helper" ? We are checking that a variable exists and converting if that's a date. Maybe finder/checker

Reply:
At the beginning I thought it can be good to have a general class which both admin / nonadmin controller will use for handling variables... That's why I call this helper... Currently they have one general thing: setting variables...
But on the other side, the same class is used now by ProcessInstanceVariableControllerImpl and ProcessInstanceControllerImpl (admin/nonadmin)...
So if this name is not proper, or it is really better to change the name,  may may do this...
Can you please provide me a good name in this case :-)